testsuite/forge/src/test_utils/consensus_utils.rs

```diff
@@ -99,14 +99,17 @@ pub async fn test_consensus_fault_tolerance(
 
         let cur = get_all_states(&validator_clients).await;
 
-        let epochs = cur.iter().map(|s| s.epoch).max().unwrap()
-            - previous.iter().map(|s| s.epoch).max().unwrap();
-        let rounds = cur
+        let (epochs, rounds) = cur
```

Comment:
if you don't have progress, but stale node sync's up, this will report progress.


I think it is probably better to take max of pairs (epoch, round), and compare those two?

Reply:
sounds good, so we want to see (3, 9) - (3, 1) in this case

Reply:
yup

Reply:
updated